configs/integration-tests.properties

```diff
@@ -2,13 +2,14 @@
 # Note that following values are just placeholders.
 # You need to provide valid configuration or all the tests will fail.
 
-tests.sdk.paServerUrl="http://localhost:8081/powerauth-java-server"
-tests.sdk.nextStepServerUrl="http://localhost:13010/powerauth-nextstep"
-tests.sdk.enrollmentServerUrl="http://localhost:8080/powerauth-webflow"
-tests.sdk.operationsServerUrl="http://localhost:8080/powerauth-webflow"
+tests.sdk.cloudServerUrl="https://your-url.com/powerauth-cloud"
+tests.sdk.cloudServerLogin="login"
+tests.sdk.cloudServerPassword="password"
+
+tests.sdk.enrollmentServerUrl="your-url.com/enrollment-server"
+tests.sdk.operationsServerUrl="your-url.com/enrollment-server"
 
 # Configuration of the PowerAuth server
-tests.sdk.appKey=""
-tests.sdk.appSecret=""
-tests.sdk.masterServerPublicKey=""
-tests.sdk.appId="1"     
\ No newline at end of file
+tests.sdk.appKey="24WxJmBGvhluACjuJMAzMw=="
```

Comment:
Are these credentials supposed to be committed to git? 
Access creds are usually excluded.

Reply:
Those are "made up" credentials for example usage, but it's probably better to have it empty to be more clear about it...

Reply:
Removed - see latest commit